cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -140,14 +135,27 @@ func (command *Evaluate) Execute(args []string) (err error) {
 
 	// Check that models and languages can be evaluated by executing the "plain" repositories.
 	log.Printf("Checking that models and languages can be used for evaluation")
+
 	// Ensure we report metrics for every model even if they are excluded.
-	assessments := report.NewAssessmentPerModelPerLanguagePerRepository(maps.Values(models), maps.Values(languages), append(command.Repositories, repositoryPlainName))
+	var assessments report.AssessmentPerModelPerLanguagePerRepository
+	{
+		models := make([]model.Model, len(command.Models))
```

Comment:
Please rename this "models" to assessmentModels" or something,  and rename the allAvailableModels->models

Then more code uses "models", this one is the exception.